src/test/streaming_test.cpp

```diff
@@ -449,6 +451,228 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+// Generative AI case + automatic timestamping server-side - Python
+
+#include <pybind11/embed.h>  // everything needed for embedding
+namespace py = pybind11;
+#include "../python/python_backend.hpp"
+
+TEST_F(StreamingTest, SingleStreamSend1Receive3Python) {
+    PythonInterpreterModule pythonModule;
+    pythonModule.start(ovms::Config::instance());
+    // Running everything in the scope, so Python objects get destroyed before module shutdown.
+    {
+        PythonBackend* pythonBackend = pythonModule.getPythonBackend();
```

Comment:
Looks to me like we could use SetUp/TearDown for L461-466 for each test (eg. with unique_ptr<> set in SetUp()). Then you shouldn't need additional scopes.